spec/facets/PartitionDatasetFacet.json

```diff
@@ -0,0 +1,63 @@
+{
+  "$schema" : "https://json-schema.org/draft/2020-12/schema",
+  "$id" : "https://openlineage.io/spec/facets/1-0-0/PartitionDatasetFacet.json",
+  "$defs" : {
+    "PartitionDatasetFacet" : {
+      "allOf" : [ {
+        "$ref" : "https://openlineage.io/spec/1-0-2/OpenLineage.json#/$defs/DatasetFacet"
```

Comment:
I think the `DatasetFacet`s typically describe the nature of the dataset, rather than the portion of the dataset being processed. I like having the partition columns described in the dataset definition, but maybe the actual partition values belong in the `InputDataset` or `OutputDataset` facets